### PR TITLE
XW-70 | add prompt to curated content modal

### PR DESCRIPTION
**Pull request by tomasznapieralski**
https://wikia-inc.atlassian.net/browse/XW-70

### CODE REVIEW

File: extensions/wikia/CuratedContent/js/CuratedContentToolModal.js [hunk 1]
```diff
@@ -27,6 +31,7 @@ define(
 		}
 
 		function close() {
+			closeModal = true;
```
Comment (bognix) [inline extensions/wikia/CuratedContent/js/CuratedContentToolModal.js]: rename to closeWithoutConfirmation

**Comment by vforge** [comment]
LGTM, except few missing `\t`

**Comment by vforge** [comment]
LGTM

File: extensions/wikia/CuratedContent/js/CuratedContentToolModal.js [hunk 2]
```diff
@@ -14,12 +15,15 @@ define(
 						size: 'medium',
 						title: title,
 						content: content
+					},
+					confirmCloseModal: function () {
+						return closeWithoutConfirmation || confirm("Are you sure you want to close this modal?");
```
Comment (bognix) [inline extensions/wikia/CuratedContent/js/CuratedContentToolModal.js]: make this string translatable

File: resources/wikia/ui_components/modal/js/modal.js [hunk 3]
```diff
@@ -31,7 +31,8 @@ define('wikia.ui.modal', [
 			vars: {
 				closeText: $.msg('close'),
 				escapeToClose: true
-			}
+			},
+			confirmCloseModal: undefined
```
Comment (bognix) [inline resources/wikia/ui_components/modal/js/modal.js]: it's undefined by default, make it false

**Comment by bognix** [comment]
LGTM

**Pull request merged**